swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -56,10 +55,10 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
     options: [String: NSObject]?,
     completionHandler: @escaping @Sendable (Error?) -> Void
   ) {
-    // Dummy start to get the extension running on macOS after upgrade
-    if options?["dryRun"] as? Bool == true {
-      Log.info("Dry run startup requested - extension awakened but not starting tunnel")
-      return completionHandler(PacketTunnelProviderError.dryStartStopCycle)
+    // Dummy start to attach a utun for cleanup later
+    if options?["dryStart"] as? Bool == true {
+      Log.info("Dry start requested - extension awakened and temporarily starting tunnel")
+      return completionHandler(nil)
```

Comment:
This ensures the tunnel _is actually started_ so that we are in a "started" state when the next IPC call hits.